hackernews/program.py

```diff
@@ -0,0 +1,45 @@
+import os
+import requests
+import time
+import urllib.parse
+
+from autokitteh.slack import slack_client
+
+
+slack = slack_client("slack_connection")
+SLACK_CHANNEL = os.getenv("SLACK_CHANNEL")
+
+
+def on_command(event):
```

Comment:
I think a short description would also be helpful that describes the functionality of this function.

What do you think about including a comment somewhere that warns the user that for low-traffic topics, it could take a while for them to receive a notification?